docs/faq.md

```diff
@@ -0,0 +1,43 @@
+
+## 4. Is Feast a data warehouse or a database?
+No. Feast does not replace your data warehouse. It orchestrates and serves data that typically resides in systems like BigQuery, Snowflake, DynamoDB, and Redis. It’s a lightweight layer to make features consistently available at training and serving time.
+
+## 5. Which problems does Feast not fully solve?
+- **Reproducible model training / experiment management**: Feast tracks feature and model metadata but doesn't version datasets or manage train/test splits. Tools like [DVC](https://dvc.org/), [MLflow](https://www.mlflow.org/), and [Kubeflow](https://www.kubeflow.org/) are more suited for that.
+- **Batch feature engineering**: Feast supports some on-demand and streaming transformations, and is investing in batch transformations, but it’s not a full-fledged batch pipeline solution.
+- **Native streaming feature ingestion**: Users can push streaming features into Feast, but Feast does not manage streaming pipelines directly (no built-in streaming pull logic).
+- **Lineage**: Although Feast ties feature values to model versions, it’s not a complete lineage solution. There are community plugins for [DataHub](https://datahubproject.io/docs/generated/ingestion/sources/feast/) and [Amundsen](https://github.com/amundsen-io/amundsen/blob/4a9d60176767c4d68d1cad5b093320ea22e26a49/databuilder/databuilder/extractor/feast_extractor.py).
+- **Data quality / drift detection**: Feast has experimental integrations with [Great Expectations](https://greatexpectations.io/) but does not provide a comprehensive solution for data drift or data quality across pipelines, labels, or model versions.
+
+## 6. Can you give examples of use cases?
+Many companies use Feast to power:
+- **Online recommendations**, leveraging pre-computed historical user/item features.
+- **Fraud detection**, by comparing new transaction patterns against historical data in real time.
+- **Churn prediction**, generating feature values for all users in batch at regular intervals.
+- **Credit scoring**, using historical features to assess default probability.
+
+## 7. How can I get started with Feast?
+- **[Quickstart](getting-started/quickstart.md)** for a hands-on intro.
+- **[Running Feast with Snowflake/GCP/AWS](how-to-guides/feast-snowflake-gcp-aws/)** for more complex setups.
+- **[Tutorials](tutorials/tutorials-overview/)** for in-depth examples and best practices.
+
+## 8. Where can I learn more about contributing?
+Check out our [Contributing Guide](project/contributing.md) for detailed instructions on how to contribute to Feast, including our coding standards, documentation practices, and release process.
+
```

Comment:
Could we add a question asking what db backends an online and offline Stores support?

Need to keep that maintained as and when we add new backends.